src/helpers/ThemeLoader.ts

```diff
@@ -0,0 +1,68 @@
+import IOverrideTheme from "../Types/IOverrideTheme"
+import * as Flex from "@twilio/flex-ui";
+import IManageThemes from "../API/IManageThemes";
+
+export default class ThemeService {
+  constructor(private _apiClient: IManageThemes) {  }
+
+  getAllThemes = async () => {
+    return await this._apiClient.getAllThemesAsync();
+  };
+
+  getUsersActiveTheme = (userId: string) => {
+    return this._apiClient.getUsersActiveThemeAsync(userId);
+  };
+
+  loadThemeForUser = (userId: string) => {
+    if (isNullOrWhitespace(userId)) return;
+
+    this._apiClient.getAllThemesAsync().then((themes) => {
```

Comment:
async/await for consistency?

Reply:
yep, have left `setThemeOverrides` as its used in `useEffect` and looks like a bit of a weird implementation for an async useEffect()